src/app/main/component/eco-news/components/news-list/news-list-gallery-view/news-list-gallery-view.component.scss

```diff
@@ -211,3 +211,23 @@ div {
     }
   }
 }
+
+@media screen and (max-width: 1000px) {
+  .list-gallery {
+    width: 285px;
+    margin-left: 7px;
+  }
+}
+
+@media screen and (max-width: 991px) {
+  .list-gallery {
+    width: 350px;
+  }
+}
+
```

Comment:
Do we really need one additional media query for a 9px difference?
Can we somehow combine these two queries into one?

Reply:
No i can't becouse from 991px to 1000px items stick together and from 1000 to .... 1900... items have another witdth same for 991 to .... 785..

Reply:
I removed all media queries except 2.
Make git pull in your branch and investigate if I broke smth.